test/integration/tenantcontroller/tenantcontroller_test.go

```diff
@@ -64,13 +64,15 @@ func setup(t *testing.T) (*httptest.Server, framework.CloseFunc, *tenantcontroll
 	networkClient := arktos.NewForConfigOrDie(configs)
```

Comment:
I think it is better to add a test case to make sure no resources are created when tenant is terminating.